ui/src/song/SongList.js

```diff
@@ -79,14 +79,14 @@ const SongList = (props) => {
       >
         {isXsmall ? (
           <SimpleList
-            primaryText={(r) => r.title}
-            secondaryText={(r) => r.artist}
-            tertiaryText={(r) => (
+            primaryText={(r) => r.title.slice(0, 35)}
```

Comment:
I'm not sure I like the truncation here: what is the rationale?

Reply:
@jvoisin if the song name is very lengthy it looks very weird so i limited the name to 35 letters you can see the issue i created

Reply:
I prefer wrapping to truncation. Let's see what @deluan prefers :)

Reply:
I also prefer wrapping